src/libs/actions/Policy.js

```diff
@@ -638,9 +650,19 @@ function updateCustomUnitRate(policyID, currentCustomUnitRate, customUnitID, new
  * @param {String} policyID
  */
 function updateLastAccessedWorkspace(policyID) {
+    if (policyID === lastAccessedWorkspacePolicyID) {
+        return;
+    }
     Onyx.set(ONYXKEYS.LAST_ACCESSED_WORKSPACE_POLICY_ID, policyID);
 }
 
+/**
+ * Delete the lastAccessedWorkspace
+ */
+function deleteLastAccessedWorkspace() {
```

Comment:
I am not sure why we need this method when we can directly call `updateLastAccessedWorkspace` ?